src/platforms/common/configuration/filtering.mdx

```diff
@@ -109,6 +109,8 @@ In its simplest form, used just for filtering the transaction, it looks like thi
 
 It also allows you to sample different transactions at different rates.
 
+If the transaction currently being processed is has a parent transaction (from an upstream service calling this service), the parent (upstream) sampling decision will always be included in the sampling context data, so that your <PlatformIdentifier name="traces-sampler" /> can choose whether and when to inherit that decision. (In most cases, inheritance is the right choice, to avoid breaking distributed traces.)
```

Comment:
```suggestion
If the transaction currently being processed has a parent transaction (from an upstream service calling this service), the parent (upstream) sampling decision will always be included in the sampling context data, so that your <PlatformIdentifier name="traces-sampler" /> can choose whether and when to inherit that decision. (In most cases, inheritance is the right choice, to avoid breaking distributed traces.)
```

Reply:
Edited @sentrivana 's suggestion with a couple of tweaks!

Reply:
Can we say anything about how the upstream sampling decision is included? Is there a property name we can mention here for users to check?

Reply:
Good point! I added a link to another page where you can see a code snippet accessing the parent sampling rate.